extension/li3ds--1.0.0.sql

```diff
@@ -72,7 +78,11 @@ create table session(
 
 create table datasource(
     id serial primary key
-    , uri varchar
+    -- possible uri schemes are "file" and "column"
+    , uri varchar not null constraint uri_scheme check (split_part(uri, ':', 1) = any ('{file, column}'::text[]))
+    , type datasource_type not null
+    , parameters jsonb
+    , capture_time timestamptz
```

Comment:
Branch updated. I replaced `capture_time` with three columns: `bounds double precision[6]`, `capture_start timestamptz` and `capture_end timestamptz`.